### PR TITLE
Update README with building instructions.

### PR DESCRIPTION
**Pull request by k3rni**
Not being a Golang developer, I had to figure out how to actually build this. There is no top-level Makefile, nor does `go build` work out of the box. So here's what I figured out, as an enumerated list section in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,18 @@ A Docker image is available:
 docker pull adrianhesketh/ddbimport
 ```
 
+## Building from source
+
+Ensure you have `$GOPATH/bin` in `$PATH` (by default that is `~/go/bin`). This is needed for statik to build the serverless package.
+Install a supported version (v12 seems to work fine) of nodejs and npm or yarn.
+
+1. `git clone git@github.com:a-h/ddbimport; cd ddbimport`
+2. `yarn global add serverless` or `npm -g install serverless`, whichever you prefer
```
Comment (k3rni) [inline README.md:37]: this needs to be run in `sls/` actually

File: README.md [hunk 2]
```diff
@@ -28,6 +28,18 @@ A Docker image is available:
...
+3. `sls plugin install -n serverless-step-functions`
```
Comment (k3rni) [inline README.md]: this command needs to be run in `sls/`

**Review by a-h — APPROVED** [review]

**Comment by a-h** [comment]
Thanks for the contribution. Much appreciated!

Just out of interest, why did you want to build from source? Did you miss that you could download binaries directly from Github releases, just a preference, or that you want to add a feature or something?

**Comment by a-h** [comment]
Ah, just seen your other PR. Makes sense now! Thanks!

**Comment by k3rni** [comment]
Indeed, I have a use case for ddbimport (which really *is* faster than any other bulk import tool I tried), that is beyond what your version provides at the moment. But my input data comes in a strange form and number of files, because of legacy. Hence the need for accepting various file sizes in the other PR.

Additionally, ddbimport, as it is now, doesn't support values of complex types, like set, list or map. I do have a patch that handles maps, but only of a very specific format (string keys, numeric values) that is of little use to others.

**Pull request merged**